tests/lib/rules/render-result-naming-convention.test.ts

```diff
@@ -0,0 +1,306 @@
+import { createRuleTester } from '../test-utils';
+import rule, {
+  RULE_NAME,
+} from '../../../lib/rules/render-result-naming-convention';
+
+const ruleTester = createRuleTester({
+  ecmaFeatures: {
+    jsx: true,
+  },
+});
+
+ruleTester.run(RULE_NAME, rule, {
+  valid: [
+    {
+      code: `
+        import { render } from '@testing-library/react';
+        
+        test('should not report straight destructured render result', () => {
+          const { rerender, getByText } = render(<SomeComponent />);
+          const button = getByText('some button');
+        });
```

Comment:
💯  very complete tests

Reply:
Thanks 😄 

I've used the test title within the code for giving each test an actual description. I'd love ESLint to provide a way for adding descriptions or titles to each tests. It's really painful to find the failing tests only from their code.